### PR TITLE
new icon: framework7 (original, original-wordmark)

### PR DESCRIPTION
**Pull request by dotarjun**
## Double check these details before you open a PR

- [x] PR does not match another non-stale PR currently opened
- [x] PR name matches the format *new icon: <i>Icon name</i> (<i>versions separated by comma</i>)*. More details [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#overview)
- [x] PR's base is the `develop` branch.
- [x] Your icons are inside a folder as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#organizational-guidelines)
- [x] SVG matches the standards laid out [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#svgStandards)
- [x] A new object is added in the `devicon.json` file as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#-updating-the-deviconjson-)

**This PR closes NONE**

#1776 
## Link to prove your SVG is correct and up-to-date.

https://framework7.io/

### CODE REVIEW

File: icons/framework7/framework-plain.svg
Comment (Gautam25Raj) [inline icons/framework7/framework-plain.svg:1]: - Use **viewBox="0 0 128 128"** 
- Remove **width="128" height="128"**.

```
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128"><path d="M.454 64c0-15.383 5.391-29.496 14.367-40.536h98.36L43.017 124.429C18.232 115.699.454 91.941.454 64zm117.94-33.11c5.809 9.659 9.152 20.992 9.152 33.11 0 35.348-28.454 64-63.546 64-4.203 0-8.312-.409-12.289-1.195zM23.199 14.934C34.238 5.614 48.469 0 64 0s29.762 5.613 40.801 14.934z"/></svg>
```

File: icons/framework7/framework-original-wordmark.svg
Comment (Gautam25Raj) [inline icons/framework7/framework-original-wordmark.svg:1]: Replace **viewBox="0 0 560 300"** with **viewBox="0 0 128 128"**  

Refer [SVG Standards](https://github.com/devicons/devicon/blob/master/CONTRIBUTING.md#svg-standards) for more information.

File: icons/framework7/framework-plain-wordmark.svg
Comment (Gautam25Raj) [inline icons/framework7/framework-plain-wordmark.svg:1]: - Replace **viewBox="0 0 1911 1024"** with **viewBox="0 0 128 128"**  
- Remove **width="1911" height="1024"**
- Optimize SVG. Use [SVG Viewer](https://www.svgviewer.dev/)

Refer [SVG Standards](https://github.com/devicons/devicon/blob/master/CONTRIBUTING.md#svg-standards) for more information.

**Review by Gautam25Raj — CHANGES_REQUESTED** [review]

**Comment by Gautam25Raj** [comment]
Maybe we don't need this plain version. Both plain and original have single color. What do you think? 🤔

**Comment by dotarjun** [comment]
agreed

**Review by lunatic-fox — APPROVED** [review]
Icons are great! ✔
Thank you for contributing! 🚀

**Review by dotarjun — COMMENTED** [review]
I keep on missing these minor intricacies haha

**Comment by Gautam25Raj** [comment]
> I keep on missing these minor intricacies haha

Same for me. 😆

**Review by Gautam25Raj — APPROVED** [review]
Great Contribution. 🚀

File: devicon.json [hunk 1]
```diff
@@ -3146,6 +3146,39 @@
...
+    {
+        "name": "framework7",
+        "altnames": [],
+        "tags": [
+            "framework",
+            "cross-platform",
+            "development",
+            "mobile",
+            "android",
+            "ios"
```
Comment (Panquesito7) [inline devicon.json:3158]:
```suggestion
            "ios",
            "open-source"
```

**Review by Gautam25Raj — APPROVED** [review]

**Review by lunatic-fox — APPROVED** [review]

**Review by Panquesito7 — APPROVED** [review]
Amazing. Thanks! 🚀

**Pull request merged**